deployment/scripts/logstash.properties

```diff
@@ -2,9 +2,9 @@
 Operator.level=INFO
 handlers=java.util.logging.ConsoleHandler,java.util.logging.FileHandler
 java.util.logging.ConsoleHandler.level=INFO
-java.util.logging.ConsoleHandler.formatter=oracle.kubernetes.operator.logging.LoggingFormatter
+java.util.logging.ConsoleHandler.formatter=oracle.kubernetes.operator.logging.OperatorLoggingFormatter
```

Comment:
Since this is a shared file, should this be OperatorLoggingFormatter? Or, is there a specific class for operator vs. webhook?

Reply:
Yes, this file is shared for operator and webhook. Currently logging format for the Operator and Webhook are similar. They both print the thread-id and the fiber name. The common logging formatter is only used in standalone command line utility. I can look into changing the webhook to common logging formatter.